fhir/resource_structure_dev/observation.json

```diff
@@ -0,0 +1,210 @@
+  },
+  "status" : "final",
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code" : "social-history",
+          "display" : "Social History"
+        }
+      ],
+      "text" : "Social History"
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://loinc.org",
+        "code" : "82810-3",
+        "display" : "Pregnancy status"
+      }
+    ],
+    "text" : "Pregnancy Status"
+  },
+  "subject" : {
+    "reference" : "Patient/<id>",
+  },
+  "encounter": {
+    "reference": "Encounter/<id>",
+  },
+  "effectiveDateTime" : "2022-08-24",
+  "valueCodeableConcept" : {
+    "coding" : [
+      {
+        "system" : "http://snomed.info/sct",
+        "version" : "http://snomed.info/sct/731000124108",
+        "code" : "77386006",
+        "display" : "Pregnant"
+      }
+    ],
+    "text" : "Pregnant"
+  }
+},
+
+{
+  "resourceType" : "Observation",
+  "id" : "example-lab-result",
+  "meta" : {
+    "profile" : [
+      "our_url_here"
+    ]
+  },
+  "extension":[
+    {
+      "url": "our-timing-extension",
+      "valueText": "during visit"
+    }
+  ],
+  "status" : "final",
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code" : "laboratory",
+          "display" : "Laboratory"
+        }
+      ],
+      "text" : "Laboratory"
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://loinc.org",
+        "code" : "718-7",
+        "display" : "Hemoglobin [Mass/volume] in Blood"
+      }
+    ],
+    "text" : "Hemoglobin"
+  },
+  "subject" : {
+    "reference" : "Patient/<id>",
+  },
+  "effectiveDateTime" : "2005-07-05",
+  "valueQuantity" : {
+    "value" : 17,
+    "unit" : "g/dL",
+    "system" : "http://unitsofmeasure.org",
+    "code" : "g/dL"
+  }
+},
+
+{
+  "resourceType" : "Observation",
+  "id" : "outcome-example",
+  "meta" : {
+    "profile" : [
+      "our_url_here"
+    ]
+  },
+  "extension":[
+    {
+      "url": "our-timing-extension",
+      "valueText": "during visit"
+    }
+  ],
+  "status" : "final",
+  "category" : [
+    {
+      "coding" : [
+        {
+          "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code" : "social-history",
+          "display" : "Social History"
+        }
+      ],
+      "text" : "Social History"
+    }
+  ],
+  "code" : {
+    "coding" : [
+      {
+        "system" : "http://loinc.org",
+        "code" : "55128-3",
+        "display" : "Discharge disposition"
```

Comment:
i'm not sure we want to have discharge in both observation and encounter - I'd suggest just encounter

Reply:
Do you think there's a good way to specifically mention the date of discharge in Encounter? Or is just having a start & end date for the encounter okay?